src/hooks.server.ts

```diff
@@ -159,6 +159,19 @@ const databaseHandle: Handle = async ({ event, resolve }) => {
     event.locals.member = member!;
   }
 
+  if (
+    i18n.route(event.url.pathname) == "/nollning" &&
+    Date.now() < Date.parse("2024-09-06T23:00:00")
+  ) {
+    redirect(302, "/nolla");
+  }
+  if (
+    i18n.route(event.url.pathname) == "/nolla" &&
+    Date.now() > Date.parse("2024-09-06T23:00:00")
+  ) {
+    redirect(302, "/nollning");
+  }
```

Comment:
```suggestion
  if (
    i18n.route(event.url.pathname) == "/nolla" &&
    Date.now() > Date.parse("2024-08-26T00:00:00")
  ) {
    redirect(302, "/nollning");
  }
```

1. I see no reason to redirect /nollning to /nolla as nolla.nu goes to /nolla
2. The date is supposed to be the first monday of the nollning (/nollning has a pre-reveal mode which still has support for ticket sales and messages)